src/redux/addCash.ts

```diff
@@ -264,6 +439,7 @@ const addCashGetTransferHash = (
         logger.log('[add cash] - add new pending txn');
         const newTxDetails = await dispatch(
           dataAddNewTransaction(txDetails),
+          //@ts-ignore
           false
```

Comment:
The `false` here wasn't allowed since `dispatch` is only defined to take one parameter. I wasn't sure, does this do anything, or can it be removed?

Reply:
Doesn't appear to be used to me — seems like a mistake. Dug into the source of redux and redux-thunk just to be sure and dispatch can only accept a single arg, even when using custom middleware.

Reply:
Makes sense, thanks!